tests/e2e-opampbridge/opampbridge/00-install.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: e2e-test-app-bridge-server
-          image: ghcr.io/open-telemetry/opentelemetry-operator/e2e-test-app-bridge-server:main
+          image: ghcr.io/open-telemetry/opentelemetry-operator/e2e-test-app-bridge-server:ve2e
```

Comment:
Where in this PR are we actually building this version? Shouldn't this be treated the same way other versions are - that is, injected via kustomize?

Reply:
we build this via the updated make command for prepare-e2e which is always run with `make prepare-e2e VERSION=e2e` resulting in this being built and loaded into kind. I initially didn't do this via Kustomize because I wasn't 100% sure how to do that (right now a lot of that is done to override the operator deployment) but let me see if i can do it for this one too.

Reply:
Maybe the CI runs that command this way, but there's no such requirement for local tests. Maybe we can just hardcode this version? Not like this image is used outside of e2e tests.